### PR TITLE
Upmerge from 1.1.x

### PR DESCRIPTION
**Pull request by nicolasnoble**
There was a few merge failures here and there, assigning relevant people.

### CODE REVIEW

**Comment by nathanielmanistaatgoogle** [comment]
Is there a description somewhere of what the manual conflicts were, where they occurred, and how they were resolved?

**Comment by nicolasnoble** [comment]
Sorry, I should've been clearer: I solved the conflicts with the relevant people, and I'm adding relevant people to that PR based on what's being touched. This is two different sets of people :-)

**Comment by nicolasnoble** [comment]
Manually solved conflicts were in `include/grpc++/impl/codegen/call.h` and `src/node/ext/call.cc` for the metadata change - the rest of the conflicts were automatically solved using `generate_projects.sh`.

**Comment by jcanizales** [comment]
image

Edit shirt to say "Why me as reviewer?" :)

**Comment by nicolasnoble** [comment]
@jcanizales feel free to change to someone else if you feel the objective-c changes aren't for you to look at.

**Comment by jcanizales** [comment]
The thing is I don't see any Objective-C changes :)

**Comment by nicolasnoble** [comment]
image
image
image

**Comment by jcanizales** [comment]
Let me rephrase that: There aren't any Objective-C nor podspec file in the list of modified files of this PR.

**Comment by murgatroid99** [comment]
What's going on with this PR?
